.doc_gen/metadata/kms_metadata.yaml

```diff
@@ -1,4 +1,47 @@
 # zexi 0.4.0

```

Comment:
java2_delete_alias is missing in the metadata

Reply:
Fixed